### PR TITLE
New configuration setting `slice_source_kwargs` 

### PR DESCRIPTION
**Pull request by forman**
Addresses #82. To close, see #86.

Checklist (strike out non-applicable):

* [x] Changes documented in `CHANGES.md`
* [x] Related issue exists and is referred to in the PR description and `CHANGES.md`
* [x] Added docstrings and API docs for any new/modified user-facing classes and functions
* [x] Changes/features documented in `docs/*`
* [x] Unit-tests adapted/added for changes/features 
* [x] Test coverage remains or increases (target 100%)

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -760,6 +760,11 @@ argument to your slice source.
     - `dict`: keyword arguments only;
     - Any other type is interpreted as single positional argument.
 
+You can also pass extra keyword arguments to your slice source using the 
+`slice_source_kwargs` setting. Its properties are passed as keyword arguments to 
+every slice source invocation. Keyword arguments passed as slice items with same name
+as in `slice_source_kwargs` take precedence.
```
Comment (thomasstorm) [inline docs/guide.md:766]: Could you please rephrase? I think I understand but I'm not sure, so I don't provide a suggestion.

File: CHANGES.md [hunk 2]
```diff
@@ -9,6 +9,11 @@
   - Introduced `SliceSource.close()` so
     [contextlib.closing()](https://docs.python.org/3/library/contextlib.html#contextlib.closing)
     is applicable. Deprecated `SliceSource.dispose()`.
+    
+  - Introduced new optional configuration setting `slice_source_kwargs` that
+    contains keyword-arguments passed to a configured `slice_source` together with
```
Comment (thomasstorm) [inline CHANGES.md:14]:
```suggestion
    contains keyword-arguments, which are passed to a configured `slice_source` together with 
```

**Review by thomasstorm — CHANGES_REQUESTED** [review]
Please address comment on guide.md

File: docs/guide.md (hunk 1 above)
Comment (forman) [reply docs/guide.md]: Fixed.

**Review by thomasstorm — APPROVED** [review]

**Pull request merged**